presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/TransformQuantifiedComparisonApplyToScalarApply.java

```diff
@@ -168,19 +182,35 @@ private PlanNode rewriteQuantifiedEqualsAllApplyNode(ApplyNode node, QuantifiedC
 
         private PlanNode rewriteQuantifiedOrderableApplyNode(ApplyNode node, QuantifiedComparisonExpression quantifiedComparison, RewriteContext<PlanNode> context)
         {
```

Comment:
Also I would prefer refactoring (variable extraction or rename) in separate commit (can be single). to not to mix the real change code with just refactoring.